test/nuxt/composables.test.ts

```diff
@@ -465,6 +477,30 @@ describe('routing utilities: `navigateTo`', () => {
     `)
     nuxtApp._processingMiddleware = false
   })
+
+  it.todo('#28425', async () => {
```

Comment:
We cant use the `.fails` here because the test results in an infinite loop that doesnt time out.